### PR TITLE
!!! BUGFIX: Adjust to TYPO3Fluid 2.5.11 and 2.6.10 signature changes

### PR DESCRIPTION
**Pull request by albe**
With this you need to update to TYPO3 Fluid 2.5.11+ or 2.6.10+

See https://github.com/TYPO3/Fluid/commit/f20db4e74cf9803c6cffca2ed2f03e1b0b89d0dc#r44244534

### CODE REVIEW

**Comment by albe** [comment]
> ERROR: UndefinedMethod - Packages/Framework/Neos.FluidAdaptor/Classes/Core/Parser/SyntaxTree/ResourceUriNode.php:52:16 - Method Neos\FluidAdaptor\Core\Parser\SyntaxTree\ResourceUriNode::rewriteBooleanNodesInArgumentsObjectTree does not exist (see https://psalm.dev/022)
        $this->rewriteBooleanNodesInArgumentsObjectTree($this->argumentDefinitions, $this->arguments);
ERROR: UndefinedMethod - Packages/Framework/Neos.FluidAdaptor/Classes/Core/Parser/SyntaxTree/ResourceUriNode.php:53:16 - Method Neos\FluidAdaptor\Core\Parser\SyntaxTree\ResourceUriNode::validateArguments does not exist (see https://psalm.dev/022)
        $this->validateArguments($this->argumentDefinitions, $this->arguments);

Still something to do...

File: Neos.FluidAdaptor/composer.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "neos/cache": "*",
     "neos/utility-files": "*",
     "neos/utility-objecthandling": "*",
-    "typo3fluid/fluid": "^2.5",
+    "typo3fluid/fluid": "~2.6.10",
```
Comment (bwaidelich) [inline Neos.FluidAdaptor/composer.json:14]:
```suggestion
    "typo3fluid/fluid": "^2.5.11 || ^2.6.10",
```

**Comment by bwaidelich** [comment]
Should be marked breaking since the two `@api` methods `AbstractViewHelper::registerArgument()` & `AbstractViewHelper::overrideArgument()` changed

**Review by bwaidelich — APPROVED** [review]
+1 by reading

File: composer.json [hunk 2]
```diff
@@ -31,7 +31,7 @@
         "neos/composer-plugin": "^2.0",
         "composer/composer": "^1.9",
         "egulias/email-validator": "^2.1",
-        "typo3fluid/fluid": "^2.5",
+        "typo3fluid/fluid": "~2.6.10",
```
Comment (albe) [inline composer.json:34]:
```suggestion
        "typo3fluid/fluid": "~2.5.11 || ^2.6.10",
```

**Comment by albe** [comment]
Post-merge addendum: Claus had a good observation on our implementation: https://github.com/TYPO3/Fluid/issues/529#issuecomment-728824665

Maybe that's the more sane way to go for 7.0?

**Pull request merged**